ddp-workspace/projects/ddp-sdk/src/lib/components/user/activities/userActivities.component.ts

```diff
@@ -237,4 +256,16 @@ export class UserActivitiesComponent implements OnInit, OnDestroy, OnChanges, Af
   private doAnalytics(action: string): void {
     this.analytics.emitCustomEvent(AnalyticsEventCategories.Dashboard, action);
   }
+  public showQuestionCount(activityInstance: ActivityInstance): boolean {
+    if (!this.config.dashboardShowQuestionCount ||
+        this.config.dashboardShowQuestionCountExceptions.includes(activityInstance.activityCode)) {
+      return false;
+    } else if (activityInstance.numQuestions === 0) {
+      return false;
+    } else if (activityInstance.statusCode === 'COMPLETE' && activityInstance.numQuestions === activityInstance.numQuestionsAnswered) {
+      return false;
+    } else {
+      return true;
+    }
+  }
```

Comment:
The logic on whether we show status name or question. I think this is not an unreasonable policy.